backend/posts/migrations/0001_initial.py

```diff
@@ -53,7 +53,7 @@ class Migration(migrations.Migration):
             },
         ),
         migrations.CreateModel(
-            name='Vote',
+            name='PostVote',
```

Comment:
For fresh database it is acceptable. But in future pls do not edit previous migrations. Create new ones that alters their behaviours.

Reply:
Tbh I'm not sure why this change was in-place because I have already kept your advice in mind.